### DISCUSSION TITLE
Is there some way to batch rename patches?

### DISCUSSION THREAD

**Discussion by alanscodelog** [stacked-git/stgit#123] [Q&A] [answered]
For example if I uncommit x number of commits, they might not have very nice names because they might be prefixed `feat/fix (scope):...` or start very similar `Fix...`, but renaming them all one by one takes a lot of time.

**Comment by topher200** [comment, marked as answer]
Wait for the next release! There's three new PRs coming that will help with this sort of workflow.

1. Add `stg rebase --interactive` command: https://github.com/stacked-git/stgit/pull/115
2. Add 'edit' instruction to `stg rebase --interactive`: https://github.com/stacked-git/stgit/pull/133
3. `stg edit` gains the power to rename a patch: https://github.com/stacked-git/stgit/pull/137

If we put these all together we can get closer to what you're looking for. Here's a screencast: 

[![asciicast](https://asciinema.org/a/424787.svg)](https://asciinema.org/a/424787)
